sources/platform/actors/limits.md

```diff
@@ -1,50 +1,134 @@
 ---
 title: Limits
-description: Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors/tasks per user or schedule.
+description: Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors and tasks per user.
```

Comment:
Learn the Apify platform's resource capability and limitations, such as max memory, disk size, and the number of actors and tasks per user.